RELEASE_NOTES.md

```diff
@@ -2,6 +2,16 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+## 1.0.3 (April 25, 2024)
+
+- 🐦 Support for Flutter 3.19.6
+- 🖨️ Improve logging for patch commands to include Flutter version being used to build
+- 💾 Improve consistency of Shorebird artifact downloads
+- ✅ Add validation that shorebird.yaml is present in pubspec.yaml assets section
+- 🌌 Improved handling of multi-dimensional Android flavors
```

Comment:
should we mention the linker debug data too? I feel that is quite a cool feature?

Reply:
Added!